sdk/nodejs/__tests__/initialize.spec.ts

```diff
@@ -0,0 +1,32 @@
+import { DVCClient, DVCCloudClient, initialize } from '../src/index'
+
+jest.mock('../src/bucketing')
+jest.mock('../src/environmentConfigManager')
+
+describe('NodeJS SDK Initialize', () => {
+    afterAll(() => {
+        jest.clearAllMocks()
+    })
+
+    it('sucessfully calls initialize with no options', async () => {
+        const client: DVCClient = await initialize('token').onClientInitialized()
+        expect(client).toBeDefined()
+    })
+
+    it('fails to initialize in Local Bucketing mode when no token is provided', async () => {
+        await expect(async () => 
```

Comment:
dont need the extra async await here

Reply:
test failed without it? 🤔

Reply:
they shouldn't, its exactly equivalent to just write a plain method that returns the promise

Reply:
we don't need the async/await outside the expect, just inside.. cuz this way, it awaits the initialize, then proceeds to check the reject.. :)

Reply:
that's not true either, if you dont await the expect then the test will pass no matter what

Reply:
I think you can actually even drop the inner function on the expect and just call initialize directly, which then passes expect the promise

Reply:
🤦 not an async function.. so don't need any async/await / rejects